package.json

```diff
@@ -119,7 +119,6 @@
     "@types/chrome-remote-interface": "0.30.0",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32862&commitId=1c3b20ea92e7f93420cebe3ea7ff946404af20af&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|18| Changed|
|Avatar Converged.size+inactive+badge.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1167| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2093| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>